usaspending_api/api_docs/api_documentation/financial_balances/agencies.md

```diff
@@ -0,0 +1,27 @@
+## [Financial Balances](#usaspending-api-documentation)
+**Route:** `/api/v2/financial_balances/agencies`
+
+**Method:** `GET`
+
+This route retrieves financial balance information by funding agency and fiscal year
+
+### Sample Request
+
+`/api/v2/financial_balances/agencies?funding_agency=775&fiscal_year=2017`
```

Comment:
Missing `_id`: should be `/api/v2/financial_balances/agencies?funding_agency_id=775&fiscal_year=2017`

Reply:
done